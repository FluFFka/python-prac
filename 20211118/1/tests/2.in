@objcount
class A(str):
    counter = 1

print(A.counter)
a = A('123123123qqq')
print(a.upper())
print(A.counter)
b = A()
print(b.counter)
del a
print(A.counter)
