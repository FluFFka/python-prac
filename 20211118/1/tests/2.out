0
123123123QQQ
1
2
1
